main/src/main/scala/org/clulab/processors/Sentence.scala

```diff
@@ -190,18 +196,19 @@ object Sentence {
     new Sentence(raw, startOffsets, endOffsets, words)
 
   def apply(
-    raw: Array[String],
-    startOffsets: Array[Int],
-    endOffsets: Array[Int],
-    words: Array[String],
-    tags: Option[Array[String]],
-    lemmas: Option[Array[String]],
-    entities: Option[Array[String]],
-    norms: Option[Array[String]],
-    chunks: Option[Array[String]],
-    tree: Option[Tree],
-    deps: GraphMap,
-    relations: Option[Array[RelationTriple]]
+             raw: Array[String],
+             startOffsets: Array[Int],
+             endOffsets: Array[Int],
+             words: Array[String],
+             tags: Option[Array[String]],
+             lemmas: Option[Array[String]],
+             entities: Option[Array[String]],
+             norms: Option[Array[String]],
+             chunks: Option[Array[String]],
+             tree: Option[Tree],
+             deps: GraphMap,
+             relations: Option[Array[RelationTriple]],
+             sections: Option[Array[String]]
```

Comment:
I can image that someone would like a default value ` = None` here so that client code won't need to change.  That's not me, but for the record...

Reply:
This one is easy. Thanks!